iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/GeneratorStartup.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Generator;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface GeneratorStartup extends Extension<Generator> {
+
+    @Override
+    default String getName() {
+        return "generatorStartup";
```

Comment:
I would change it back to `startup` to be consistent with other extension name (see `LoadDetail` for example). The extension name should not contain the equipment.

Reply:
Be careful to keep compat with existing data